src/modules/85-cv/components/ExecutionVerification/components/LogAnalysisContainer/components/LogAnalysisRadarChart.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useCallback, useMemo } from 'react'
+import { Container, Icon, NoDataCard, Layout, Text } from '@wings-software/uicore'
+import { Color, FontVariation } from '@harness/design-system'
+import Highcharts from 'highcharts'
+import HighchartsReact from 'highcharts-react-official'
+import Boost from 'highcharts/modules/boost'
+import { useStrings } from 'framework/strings'
+import type { LogAnalysisRadarChartProps } from './LogAnalysisRadarChart.types'
+import MultiRangeSlider from './MinMaxSlider'
+import getLogAnalysisSpiderChartOptions, { getRadarChartSeries } from './LogAnalysisRadarChart.utils'
+import styles from '../LogAnalysis.module.scss'
+
+Boost(Highcharts)
+
+const LogAnalysisRadarChart: React.FC<LogAnalysisRadarChartProps> = ({
+  clusterChartLoading,
+  clusterChartData,
+  handleAngleChange,
+  filteredAngle,
+  onRadarPointClick
+}) => {
+  const radarChartSeries = getRadarChartSeries(clusterChartData?.resource || [])
+
+  const filteredSeries = useMemo(
+    () =>
+      radarChartSeries?.filter(
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore
+        data => data?.data[0].x >= filteredAngle.min && data?.data[0].x <= filteredAngle.max
+      ),
+    [filteredAngle, radarChartSeries]
+  )
+
+  const { getString } = useStrings()
+
+  const handleRadarPointClick = useCallback((pointClusterId: string) => {
+    onRadarPointClick(pointClusterId)
+  }, [])
```

Comment:
Codacy has a fix for the issue: [React Hook useCallback has a missing dependency: 'onRadarPointClick'. Either include it or remove the dependency array. If 'onRadarPointClick' changes too often, find the parent component that defines it and wrap that definition in useCallback.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232318)
```suggestion
  }, [onRadarPointClick])
```